datasets/bbc_hindi_nli/README.md

```diff
@@ -0,0 +1,207 @@
+---
+annotations_creators:
+- machine-generated
+language_creators:
+- found
+languages:
+- hi
+licenses:
+- mit
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- extended|bbc__hindi_news_classification
+task_categories:
+- text-classification
+task_ids:
+- natural-language-inference
+---
+
+# Dataset Card for BBC Hindi NLI Dataset
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- HomePage : https://github.com/midas-research/hindi-nli-data
+- Paper : "https://www.aclweb.org/anthology/2020.aacl-main.71"
+- Point of Contact : https://github.com/midas-research/hindi-nli-data
+
+### Dataset Summary
+
+- Dataset for Natural Language Inference in Hindi Language. BBC Hindi Dataset consists of textual-entailment pairs.
+- Each row of the Datasets if made up of 4 columns - Premise, Hypothesis, Label and Topic.
+- Context and Hypothesis is written in Hindi while Entailment_Label is in English.
+- Entailment_label is of 2 types - entailed and not-entailed.
+- Dataset can be used to train models for Natural Language Inference tasks in Hindi Language.
+[More Information Needed]
+
+### Supported Tasks and Leaderboards
+
+- Natural Language Inference for Hindi
+
+### Languages
+
+Dataset is in Hindi
+
+## Dataset Instances
+
+<table>
+<tbody>
+<tr>
+<td>&nbsp;Premise</td>
+<td>&nbsp;Hypothesis</td>
+<td>&nbsp;Label</td>
+<td>&nbsp;Topic</td>
+</tr>
+<tr>
+<td>&nbsp;बम धमाकों से दहला इराक, दर्जनों मरे</td>
+<td>&nbsp;यह भारत की सूचना है|</td>
+<td>&nbsp;not-entailed</td>
+<td>&nbsp;2</td>
+</tr>
+<tr>
+<td>&nbsp;बर्मा में मुसलमानों के खिलाफ हिंसा के और सुबूत</td>
+<td>&nbsp;यह भारत की सूचना नहीं है|</td>
+<td>&nbsp;entailed</td>
+<td>&nbsp;2</td>
+</tr>
+</tbody>
+</table>
```

Comment:
Moreover we expect an actual example as it is when you use the dataset in python.
You can use this for example:

````suggestion
```
{'hypothesis': 'यह खबर की सूचना है|',
 'label': 'entailed',
 'premise': 'गोपनीयता की नीति',
 'topic': '1'}
```
````

It's the first example of the train set